source/usage-examples/changeStream.txt

```diff
@@ -0,0 +1,56 @@
+=================
+Watch for Changes
+=================
+
+.. default-domain:: mongodb
+
+Open a Change Stream
+--------------------
+
+You can keep track of changes of data in MongoDB, such as changes to a
+collection, database, or deployment by opening a :manual:`Change
+Stream</changeStreams/>`. A ``Change Stream`` allows applications to
+watch for changes to data and react to those changes. You can open a
+``Change Stream`` by calling the :node-api:`watch()
+</Collection.html#watch>` method on a collection, database, or
+deployment. 
+
+The ``watch()`` method takes an optional pipeline as the first parameter
+that is an array of stages to filter the :manual:`change events
+</reference/change-events/>` output. Also, ``watch()`` can take in
+additional options object as the second parameter. Set ``fullDocument``
+field of the additional options object to ``'updateLookup'`` to receive
+an event that contains the changes to the document as well as the entire
+altered document. Alternatively, you can leave  ``fullDocument`` or set it to the string ``'default'`` to only receive the altered document, but not the changes to the document. 
+
+Process the Change Stream Events
+--------------------------------
+
+You can capture events from a change stream instance by using a listener
+function. Call the watch() command to get a Change Stream instance. Add
+your listener function by calling the `EventEmitter.on()
+<https://nodejs.org/api/events.html#events_emitter_on_eventname_listener>`_
+method on that instance. Set the value of the first parameter to the
+string ``'change'`` and add your :mdn:`callback function
+<Glossary/Callback_function>` function as the second parameter.  The
+callback will trigger when a change event is emitted, providing the next
+available document. You can use the callback to specify logic to process
```

Comment:
Nitpick: This can be more direct and succinct by omitting the general statement "can use the callback". E.g.
"You can specify logic in the callback to process the event document when it is received."

Reply:
Simple but clear wording change - thanks, fixed verbatim